pages/visit/why-abu-dhabi.jsx

```diff
@@ -2,20 +2,48 @@ import { Container } from "@mui/material";
 
 import EvLayout from "components/layouts/EvLayout";
 import api from "utils/api/grocery3-shop";
+import Section1 from "../../src/pages-sections/why-abu-dhabi/Section1";
+import Section2 from "../../src/pages-sections/why-abu-dhabi/Section2";
+import Goal from "../../src/pages-sections/why-abu-dhabi/Goal";
+import SponsorsGrid from "../../src/pages-sections/why-abu-dhabi/SponsorsGrid";
+import PartnersGrid from "../../src/pages-sections/why-abu-dhabi/PartnersGrid";
 // ======================================================
 // ======================================================
 const generalPage = (props) => {
   const { offerProducts, allProducts, topSailedProducts } = props;
 
   return (
     <EvLayout showNavbar={true}>
+      <Section1 />
       <Container
         sx={{
           mb: 6,
         }}
       >
-        <div>why-abu-dhabi</div>
+        <Section2 />
+        <Goal imageSrc={"/assets/images/why-abu-dhabi/sustainability.jpg"} headline={"Sustainability"} paragraph={`Abu Dhabi Urban Planning Council (UPC). Abu Dhabi’s Plan 2030 establishes a clear vision for sustainability as the foundation of any new development occurring in the Emirate of Abu Dhabi. The purpose of Estidama is to create a new sustainability framework that will direct the current course of development while allowing adaptation as new understanding evolves. Its ultimate goal is to create an ever-improving quality of life for the residents of the Emirate on four equal pillars of sustainability: environmental, economic, social, and cultural.`} />
```

Comment:
separate data and presentation here -> use an array of json objects to keep the data of each goal and use Array.map() to pass it to the Goal components